name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/ibflex/IBFlexStatementExtractor.java

```diff
@@ -263,16 +276,35 @@ else if (this.ibAccountCurrency.equals(s.getCurrencyCode()))
             setAmount(element, transaction, amount);
 
             // Set note
-            transaction.setNote(element.getAttribute("description"));
+            if (!transaction.getType().equals(AccountTransaction.Type.DIVIDENDS))
```

Comment:
@Nirus2000 : What's the reason for this change (not recording original description for dividend transaction) ? Nothing in the commit message explains that. IMHO, this description is one of the useful which could be recorded, as it includes high-precision dividend per share, type of dividend and even ISIN (not easily seen in IBKR data otherwise).

Reply:
@Nirus2000 : Thanks for promptly looking into this!

> but you are the first of many thousands of users to notice this. 

Well, that's fairly recent change for many to try it. Then not everybody will notice it (at least for a while). Then of those who notice it, only a few will report. I for one wanted to as about it for a week and kept fogeting...

> The importer was revised because it did not include many new features.

Sure, that work is much appreciated. My question was specifically why `if (!transaction.getType().equals(AccountTransaction.Type.DIVIDENDS))` line was added, i.e. why description was started to not be recorded for dividends and only dividends.